bolt-anything.bash

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
+
+## STAGE 1 = build llvm-bolt
+## STAGE 2 = Instrument binary to run a workload with it to gather profiles for optimizing
+## STAGE 3 = Merging the created data to one file and remove the not needed data## Actually bug in llvm-bolt https://github.com/llvm/llvm-project/issues/56209
+## STAGE 4 = Optimize the binary with the created profile
+STAGE=
+
+## File or binary you want to instrument and then bolt
+BINARY=libLLVM-14.so
+
+## PATH to the target
+BINARYPATH=/usr/lib
+
+## PATH where llvm-bolt is
+BOLTPATH=~/toolchain/llvm/llvm-bolt/bin
+
+## BASEDIR for data
+TOPLEV=~/toolchain/bolt
+
+## Here can be the optimized binarys,  merged fdata and your original binary/file as backup
+BOLTBIN=${TOPLEV}/bin
+
+## PATH FOR INTRUMENTED DATA
+## Use a own PATH for it since it creates alot of files
+FDATA=${TOPLEV}/fdata
+
+
+################################################################
+################################################################
+################################################################
+################################################################
+
+
+create_path() {
+    ## Create PATH's
+    mkdir -p ${FDATA}
+    mkdir -p ${BOLTBIN}
+}
+
+check_requirements() {
```

Comment:
is this used?

Reply:
Actually not, no. I will remove it. It did not worked like I thought.